docs/Examples.md

```diff
@@ -729,3 +738,78 @@ spec:
         secretName: splunk-cluster-master
   defaultsUrl: /mnt/cluster-master/default.yml
 ```
+## Managing global kubernetes secret object
+
+### Creating global kubernetes secret object
+
+To create the global kubernetes secret object you can use the kubectl command line tool. Here are the steps to the same:
+
+1. Retrieve the namespace in the current context using the following command `kubectl config view --minify --output 'jsonpath={..namespace}'` and make a note of it.
+If the command doesn't display an output it indicates that we are in the `default` namespace.
+
+2. Gather the values for all splunk secret tokens you want to configure on the global kubernetes secret object. All possible Splunk secret tokens along with their corresponding key strings in the global kubernetes secret object are mentioned [here](PasswordManagement.md#splunk-secret-tokens-in-the-global-secret-object).
+
+3. Create a kubernetes secret object whose name is of the format splunk-<namespace_retrieved_above>-secret using the command below. In this example we are creating the global kubernetes secret object with default administrator password and pass4symmkey configured.  
+`kubectl create secret generic splunk-<namespace_retrieved_above>-secret --from-literal='password=<admin_password_value>' --from-literal='pass4symmkey=<pass4symmkey_value>'`
+
+### Reading global kubernetes secret object
+
+Once created, all secret tokens in the secret object are base64 encoded. To read the global kubernetes secret object you can use the following command:
+
+`kubectl get secret splunk-default-secret -o yaml`
```

Comment:
should it be `splunk-<namespace_retrieved_above>-secret` ?

Reply:
Done.